lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -23,8 +23,10 @@ use OpenQA::Schema::Result::JobDependencies;
 use OpenQA::Utils qw(determine_web_ui_web_socket_url get_ws_status_only_url);
 use Mojo::ByteStream;
 use Mojo::Util 'xml_escape';
+use Mojo::File 'path';
 use File::Basename;
 use POSIX 'strftime';
+use File::Spec::Functions qw(catdir);
```

Comment:
I wouldn't use that function because the code is using `Mojo::File::path` anyways which provides the same feature.

Reply:
i will do

Reply:
removed. I replaced it with the `Mojo::File::path`